Cargo.toml

```diff
@@ -11,3 +11,9 @@ tokio = { version = "1.34.0", features = ["full"] }
 sqlx = { version = "0.7.2", features = ["postgres","runtime-tokio"] }
 serde = { version = "1.0.192", features = ["derive"] }
 serde_json = "1.0.108"
+env_logger = "0.10.1"
+logging = "0.1.0"
```

Comment:
i don't think important the correct [logger](https://crates.io/crates/logging)

and `env_logger` is also not used
I believe the functionality of these crates is included in `tracing`